src/ray/gcs/callback.h

```diff
@@ -1,6 +1,10 @@
 #ifndef RAY_GCS_CALLBACK_H
 #define RAY_GCS_CALLBACK_H
 
+#if defined(__clang__) && defined(_MSC_VER)
+// TODO: Remove this Windows (clang-cl) workaround once we upgrade to Booost > 1.68: https://lists.boost.org/Archives/boost/2018/09/243420.php
```

Comment:
@mehrdadn for the TODOs, can you do `TODO(mehrdad)` or `TODO(mehrdadn)` (or whatever abbreviation you prefer)? The reason is so that if someone has a question about it in the future, they know who to ask.

Reply:
Sure, done.